src/count.js

```diff
@@ -20,6 +20,9 @@ function count(ast) {
     case 'keyframes':
     case 'import':
     case 'supports':
+    case 'charset':
+    case 'namespace':
+    case 'page':
```

Comment:
Gotcha, I pushed an update to reflect that, so `return 1`, will count `@page` as 1 selector and should not be added after the 4095th selector, is that right?